javaparser-core/src/main/java/com/github/javaparser/jctree/ast/AJCAnnotation.java

```diff
@@ -0,0 +1,38 @@
+package com.github.javaparser.jctree.ast;
+
+import com.sun.tools.javac.tree.JCTree.*;
+import com.sun.tools.javac.tree.JCTree;
+import com.sun.tools.javac.util.List;
+//import com.sun.tools.javac.util.Name;
```

Comment:
Do you mean to remove the commented "imports"?

No problem, I kept them so that, if I had to make massive changes into the files, all of them had a header with same number of lines.